requirements.txt

```diff
@@ -1,8 +1,11 @@
-marshmallow==3.0.0rc6
 argschema==2.0.1
 numpy
 pillow
 scikit-image
 matplotlib
-opencv-python
+opencv-python==4.4.0.42
```

Comment:
I wonder if it might make more sense to set an upper limit (`opencv-python<5.0`) instead of pinning explicitly?

Reply:
addressed in earlier PR